cg-spec/editors_draft.html

```diff
@@ -404,11 +407,23 @@ <h2>Triples and occurrences</h2>
+
+      <p>Referring to a specific occurrence, not to the triple as an abstract entity, is a common use case: provenance annotations on occurrences have until recently been the go-to example for RDF-star, as witnessed by the so-called <a href="#the-seminal-example">seminal example</a>. Attributed relations as in <a href="https://w3c.github.io/rdf-star/UCR/rdf-star-ucr.html#support-wikidata-wikibase-data-model">WikiData</a> or Property Graphs and also n-ary relations commonly refer to occurrences. It is important to be aware that when using RDF-star embedded triples such statement annotations have to provide references to both the occurrence and its containing set or graph or source as extra statements to be correct and complete, thereby putting a certain limit to the perceived simplicity and elegance of the RDF-star approach. Querying in SPARQL consequently also becomes more involved.</p>
+
+    <p class="issue" data-number="">The section <a href="#the-seminal-example">The seminal example</a> should be moved here as this is the context in which it is relevant and can be best understood.</br>
+      I'm leaving it below for now to make it easier to follow the changes that I made to that section in DIFF view.</p>
 
   </section>
 
-  <section>
+</section>
+
+<section>
+  <p class="issue" data-number="">The section <a href="#rdf-star-#semantics">RDF-star Semantics</a> should be moved here, after "Concepts and Abstract Syntax" and before "Concrete Syntaxes ". This better reflects the importance of the consequences that the proposed extension of RDF semantics may have. It also fits better into the flow of the report as right now we have 1. Introduction, 2. RDF, 3. RDF, 4. SPARQL, 5. SPARQL, 6. RDF. It also concentrates the controversial topics in the first 3 sections, making it easier for the reader to follow the discussion. Putting Semantics last risks to hide the important issues involved.</br>
```

Comment:
I agree that it is strange to have two sections on SPARQL between RDF-related sections. So I am in favor of moving the semantic up.

My intuition, though, is that more people are interested in the concrete syntaxes than in the formal semantics, which was the reason to put the former first, and the latter second.

Reply:
I don't want to sound like I'm not willing to compromise. I'd certainly be glad enough if "Semantics" would be moved up before SPARQL. I still think that they should follow directly on the "Concepts and Abstract Syntax" section. People will not be shy to skip them to get to the syntaxes section ;-)